content/terraform-docs-common/docs/cloud-docs/integrations/service-now/service-catalog-terraform/example-customizations.mdx

```diff
@@ -151,6 +151,17 @@ the default Workspace Variables.
    the Flow then click the **Update**.
    ![Screenshot: ServiceNow Update Process Engine](/img/docs/servicenow-catalog-update-process-engine.png "Screenshot of updating Process Engine for the Catalog Item")
 
+## Adjusting Polling Schedule Intervals
+
+Customers can customize the frequency at which Terraform-related updates are retrieved by adjusting polling schedule intervals.
+There are three configurable polling intervals: 
+1. **Worker Poll Apply Run**
+1. **Worker Poll Destroy Workspace**
+1. **Worker Poll Run State** 
+
+To modify these intervals, navigate to **All > Flow Designer > Flows** in ServiceNow. You may modify this frequency by changing
+the associated flow's Scheduled Trigger, allowing for more or less frequent updates in the ServiceNow ticket. 
```

Comment:
Bolding because I assumer this is in the UI?
```suggestion
the associated flow's **Scheduled Trigger**, allowing for more or less frequent updates in the ServiceNow ticket. 
```

Reply:
yup